sources/packages/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -26,3 +26,13 @@ export interface StudentApplicationDTO {
   award: string;
   status: string;
 }
+
+export interface PIRSummaryDTO {
+  applicationNumber: string;
```

Comment:
applicationNumber need not be type number? any particular reason?

Reply:
usually, long numbers like definite characters, it's better to use string, eg. '080' vs number 080  is different

Reply:
I am okay for a string, but the number in TS should be big enough to handle 16 characters I guess, variable name as applicationNumber got me confused. @andrewsignori-aot , can you please help me out on the understanding. We can discuss.

Reply:
it's not about the number of characters @guru, 
1. the string '080' saved in DB will always be '080'. 080 int/number saved in DB will be 80. 
2. '080' and '80' strings are different always

Reply:
okay got it